universal-application-tool-0.0.1/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -159,6 +167,58 @@ public ApplicantProgramBlocksController(
             });
   }
 
+  @Secure
+  public CompletionStage<Result> goToPreviousPage(
```

Comment:
Question (blocking): why have both this method and `previous`?

Reply:
Good catch - meant to clean this up! Consolidated into `previous`.